contact/forms.py

```diff
@@ -0,0 +1,67 @@
+from django import forms
+
+
+class ContactForm(forms.Form):
```

Comment:
Contact Us form moved out of admin app and into a Django form, double checked with Rusty.  Existing form was just raw html.  It would have been difficult to add the crsf token, recaptcha keys, and UI updates if recaptcha failed to the existing html. I explored `wagtailforms` so community could add form fields themselves, but this seemed excessive effort for a form that's not likely to change often.